q0334/q0334.cpp

```diff
@@ -0,0 +1,20 @@
+class Solution {
+public:
+    bool increasingTriplet(vector<int>& nums) {
+        long long minFirst, minSec;

```

Comment:
I think if the input array is a vector of integers, it is probably fine to use int to track these variables

Reply:
@chunyuema yeah but since long long is used for the max and min values in the following part, I need to declare them as long long here too I think. 
` minFirst = numeric_limits<long long>::max();
        minSec = numeric_limits<long long>::max() - 1;`
Otherwise, it gives a wrong answer for the following case,
[2147483646,2147483647,2147483647]
Please let me know further thoughts.

Reply:
INT_MAX could work. But since the nums[i] range is -2^31 <= nums[i] <= 2^31 - 1, which is the range for integers, I needed to use long long for accommodating something >2^31 - 1 or <-2^31 - 1. Sorry, my explanation might be bad.